website/content/api-docs/system/inspect/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+layout: api
+page_title: /sys/internal/inspect - HTTP API
+description: >-
+  This endpoint is intended to inspect a specific internal subsytem for debugging purposes. It can be accessed with a root token or sudo priveleges.
+---
+
+# `/sys/internal/inspect`
+
+The `/sys/internal/inspect` is intended to inspect a specific internal subsystem for debugging purposes.
+It can be accessed with a root token or sudo privileges.
```

Comment:
I think it would be good to set expectations about who is intended to use this endpoint (interpreting output will require some familiarity with the Vault codebase and will not immediately point to obvious issues).

Reply:
Added this information in the current commit!

Reply:
I'm ok leaving the language as-is. I tend to think of our "admin" persona as someone who doesn't necessarily do a ton of application development, but is in more of either an "ops admin" or "compliance/iam admin" type of role. I'm not sure whether most people in those roles would be able to draw conclusions about what's wrong with their Vault instance from this API.